doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,83 @@
+.. procfs-plugin:
+
+======================================================================
+Using the ProcFS plugin
+======================================================================
+
+
+Implemented in |Percona Server| 8.0.25-15, the ProcFS plugin provides access to the Linux performance counters by running SQL queries against a Percona Server for MySQL 8.0 (or other MySQL 8.0-compatible variant). This plugin is useful for Cloud and MySQL-as-a-service installations.
```

Comment:
Can you please provide more information on this
> This plugin is useful for Cloud and MySQL-as-a-service installations

How it's useful?

Reply:
Added information - basically, it allows a user to access metrics when they cannot access the operating system.